go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/EasyPost/easypost-go/v2
 
-go 1.15
+go 1.16
 
 require (
 	github.com/dnaeon/go-vcr v1.2.0
 	github.com/google/go-querystring v1.1.0
-	github.com/google/uuid v1.3.0 // indirect
-	github.com/stretchr/testify v1.7.2
-	golang.org/x/text v0.3.8
+	github.com/google/uuid v1.3.0
+	github.com/stretchr/testify v1.8.2  // last version supporting < Go 1.19
```

Comment:
Thought: You won't want this comment, only because go's tools will automatically overwrite this when run. Maybe try the line above it or remove it entirely